client/src/app/core/ui-services/login-data.service.ts

```diff
@@ -29,6 +36,18 @@ const LOGIN_DATA_STORAGE_KEY = 'LoginData';
     providedIn: 'root'
 })
 export class LoginDataService {
```

Comment:
@FinnStutzenstein 
There seems to be an issue with the `ConfigService` and this `LoginDataService` As described in the review, hard-reloading leaves the `general_event_privacy_policy` and the `general_event_legal_notice` empty.